### PR TITLE
Add ratio display to new design

### PR DESCRIPTION
**Pull request by aaronkerckhoff**
Since I got selected for the YouTube A/B-Testing I got the new design. I decided to add the like-dislike-ratio display also to the new design.

### CODE REVIEW

**Comment by aaronkerckhoff** [comment]
> Hi @aaronkerckhoff can you take a look at #495? It seems related.

Looks interesting, but I think this is not related to my update. The user recommends using normal HTML, but my update only adds the current feature to the new design. If anyone updates the way `createRateBar` works this will also automatically apply to my current change. Maybe I'll do this in another commit soon, but currently, I'm only changing that the bar also gets rendered on the new theme.

**Comment by aaronkerckhoff** [comment]
I think there are mainly two problems with the YouTube-HTML:
1. Custom color schemes and designs are probably more difficult to implement
2. If YouTube decides to remove the unused HTML for some reason in the future, the extension will stop working.

File: Extensions/combined/src/state.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ function isShorts() {
   return location.pathname.startsWith("/shorts");
 }
 
+function isNewDesign() {
+  return document.getElementById("comment-teaser") !== null;
```
[the remark this answers is not in the document]
Comment (aaronkerckhoff) [reply Extensions/combined/src/state.js:43]: I tried it with a live stream and it worked. It was just hidden, but existent in the HTML. I don't find any premieres or future-timed videos at the moment. If you know any, lmk.

**Comment by aaronkerckhoff** [comment]
> does it fix these issues? #481 #522 #651

#481 My additions don't fix this, but I don't have the bug mentioned even though I have the new theme. Maybe something in YouTube-HTML changed and it works again.
#522 Yes, my PR fixes this
#651 No, this is a different issue

**Comment by Anarios** [comment]
Thanks!

**Pull request merged**